python/delta_sharing/tests/test_delta_sharing.py

```diff
@@ -710,6 +710,10 @@ def test_load_as_pandas_success_dv_and_cm(
     expected['partition_col'] = expected['partition_col'].astype('int32')
     pd.testing.assert_frame_equal(pdf, expected)
 
+    # Test client specifying explicit delta format
+    pdf = load_as_pandas(f"{profile_path}#{fragments}", limit, version, None, None, True)
```

Comment:
did you have output to ensure that it is using the new code path?

Reply:
yup i logged it and it is using the new path

Reply:
o, and could you also hack the profile file and test against our production server to verify the server respects this config?